web/src/App.jsx

```
@@ -5253,7 +5264,23 @@ function App() {
+                    <p style={{ maxWidth: '400px', textAlign: 'center', color: 'var(--muted)', marginBottom: '24px' }}>
+                      Your messages will appear here once your phone syncs.
+                      Open PulseLink on your Android device to start the sync.
+                    </p>
+                    <button
+                      className="primary-btn"
+                      onClick={requestPhoneSync}
+                    >
+                      Request Sync from Phone
+                    </button>
+                    {syncRequestStatus && <div className={getToastClass(syncRequestStatus)} style={{marginTop: 16}} role="status" aria-live="polite">{syncRequestStatus}</div>}
+                  </div>
```

Biome (2.3.13)

[error] The elements with this role can be changed to the following elements:
<output>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)